config/keys.go

```diff
@@ -208,4 +208,5 @@ const (
 	TracingExporterTypeKey                             = "tracing-exporter-type"
 	TracingHeadersKey                                  = "tracing-headers"
 	ProcessContextFileKey                              = "process-context-file"
+	ProposerMinBlockDelayKey                           = "proposer-min-block-delay"
```

Comment:
Also wondering if this is only supposed to impact the primary network - if we should name it something like `--primary-network-min-block-delay`